Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/functionapp/AzureFunctionAppMvpModel.kt

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright (c) 2018 JetBrains s.r.o.
```

Comment:
I think it's time to update the copyright year :)

Reply:
Sure :) Updated year for all new files. I'll make a separate commit with already existing files.